machine/wnc/wnc_model/busybox/patches/onie-syseeprom-wnc_modify.patch

```diff
@@ -0,0 +1,174 @@
+diff --git a/miscutils/sys_eeprom.c b/miscutils/sys_eeprom.c
+index 3003bd6..baf5094 100644
+--- a/miscutils/sys_eeprom.c
++++ b/miscutils/sys_eeprom.c
+@@ -2,6 +2,26 @@
+ #include "onie_tlvinfo.h"
+ #include <getopt.h>
+ 
++/* refer with celestica i2c-tool patch.*/
++static const struct tlv_code_desc tlv_code_default[] = {
++    { TLV_CODE_PRODUCT_NAME	 , "WNC-X86-ONIE"},
++    { TLV_CODE_PART_NUMBER	 , "xxxxx-yyyyy-zz"},
++    { TLV_CODE_SERIAL_NUMBER , "WNC0000000001"},
++    { TLV_CODE_MAC_BASE	     , "00:00:00:00:00:01"},
++    { TLV_CODE_MANUF_DATE	 , "04/15/2015 14:30:00"},//MM/DD/YYYY hh:mm:ss
++    { TLV_CODE_DEVICE_VERSION, "0"},
++    { TLV_CODE_LABEL_REVISION, "WNC-X86-ONIE-REV_0"},
++    { TLV_CODE_PLATFORM_NAME , "WNC-X86-ONIE"},
++    { TLV_CODE_ONIE_VERSION	 , "2014.04"},
++    { TLV_CODE_MAC_SIZE	     , "73"},
++    { TLV_CODE_MANUF_NAME	 , "WNC"},
++    { TLV_CODE_MANUF_COUNTRY , "TW"},
++    { TLV_CODE_VENDOR_NAME	 , "XXX"},
++    { TLV_CODE_DIAG_VERSION	 , "1.0.0"},
++    { TLV_CODE_SERVICE_TAG   , "LB"},
++    { TLV_CODE_VENDOR_EXT	 , "66"},
++};
++
+ static u_int8_t eeprom[SYS_EEPROM_SIZE];
+ /*
+  *  This macro defines the onie-syseeprom command line command.
+@@ -22,7 +42,7 @@ cmd_usage()
+ {
+     static const char *usage =
+ 	"Display and program the system EEPROM data block.\n"
+-	"Usage: onie-syseeprom [-h][-l] [-e] [-s <code>=<value>,...]\n"
++	"Usage: sys-eeprom [-h][-l] [-e] [-s <code>=<value>,...]\n"
+ 	"   With no arguments display the EEPROM contents.\n"
+ 	"   -h --help\n"
+ 	"      Display usage\n"
+@@ -30,11 +50,15 @@ cmd_usage()
+ 	"      List the understood TLV codes and names.\n"
+ 	"   -e --erase\n"
+ 	"      Reset the EEPROM data.\n"
++	"   -r --restore\n"
++	"      Restore the EEPROM data to default.\n"
+ 	"   -g --get <code>\n"
+ 	"      Look up a TLV by code and write the value to stdout.\n"
+ 	"   -s --set <code>=<value>,<code>=<value>...\n"
+ 	"      Set a TLV code to a value.\n"
+-	"      If no value, TLV is deleted.\n";
++	"      If no value, TLV is deleted.\n"
++	"   -d --dump\n"
++	"      Dump the EEPROM TLV data.\n";
```

Comment:
The `--dump` option adds no value.  The default behaviour with no args is to dump the EEPROM already.

Having a onie-syseeprom command that behaves differently from everyone else will cause problems for end users.